pkgs/development/libraries/vapoursynth/default.nix

```diff
@@ -36,12 +41,17 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  passthru = {
+  passthru = rec {
     # If vapoursynth is added to the build inputs of mpv and then
     # used in the wrapping of it, we want to know once inside the
     # wrapper, what python3 version was used to build vapoursynth so
     # the right python3.sitePackages will be used there.
     inherit python3;
+
+    withPlugins = import ./plugin-interface.nix {
+      inherit lib python3 buildEnv writeText concatMapStringsSep runCommandCC
```

Comment:
`concatMapStringsSep` is inside `lib`, you can avoid passing it separately if you use `lib.concatMapStringsSep`.

Reply:
Thanks. I fixed it.